test/erlcloud_ddb2_tests.erl

```diff
@@ -4091,3 +4103,852 @@ update_time_to_live_output_tests(_) ->
                 enabled = true}}})],
     output_tests(?_f(erlcloud_ddb2:update_time_to_live(<<"SessionData">>, 
       [{attribute_name, <<"ExpirationTime">>}, {enabled, true}])), Tests).
+
+%% ListBackups test based on the API examples:
+%% https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ListBackups.html
+list_backups_input_tests(_) ->
+ Tests =
+  [?_ddb_test(
+   {"ListBackups example request",
+    ?_f(erlcloud_ddb2:list_backups([{limit, 4},{exclusive_start_backup_arn, <<"arn:aws:dynamodb:">>},{table_name, <<"Forum">>},{time_range_lower_bound,1522926603.688},{time_range_upper_bound, 1523022454.098}])), "
+    {
+        \"Limit\": 4,
+        \"TableName\": \"Forum\",
+        \"ExclusiveStartBackupArn\": \"arn:aws:dynamodb:\",
```

Comment:
I'd personally prefer a made-up ARN here that actually mimics the ARN format for table backups.